JWS selinux step is incomplete

##### SUMMARY
The selinux step in `roles/jws/tasks/systemd/selinux.yml` builds and installs the policy correctly, but it doesn't allow acces to the ports with semanage as outlined in the documentation. Any ports being used by tomcat should be labeled as `http_port_t` with `semanage port -a -t http_port_t -p tcp <port>`.

**Note:** The step is only required if you use a port that isn't covered by `http_port_t`, default values (on RHEL 8) are:

```
# semanage port -l | grep ^http.*_port_t
http_cache_port_t              tcp      8080, 8118, 8123, 10001-10010
http_cache_port_t              udp      3130
http_port_t                    tcp      80, 81, 443, 488, 8008, 8009, 8443, 9000
```


##### ISSUE TYPE
 - Bug Report


##### ANSIBLE VERSION
<!-- Paste, BELOW THIS COMMENT, verbatim output from "ansible --version"-->
```
ansible 2.9.27
  config file = /home/csutherl/source/jws-ansible-playbook/ansible.cfg
  configured module search path = ['/home/csutherl/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.9.12 (main, Mar 25 2022, 00:00:00) [GCC 11.2.1 20220127 (Red Hat 11.2.1-9)]
```

##### COLLECTION VERSION
<!-- Paste, BELOW THIS COMMENT, verbatim output from "ansible-galaxy collection list"-->
Dev build of the collection based on commit 87238cc4.

##### STEPS TO REPRODUCE
<!-- List the steps to reproduce the problem, using a minimal test-case. -->
Install JWS using `tomcat_listen_http_port: 8567` (not in `http_port_t` or any other port_t by default) and observe the log.

##### EXPECTED RESULTS
<!-- What did you expect to happen when running the steps above? -->
The Connector successfully starts and is accessible.

##### ACTUAL RESULTS
<!-- What actually happened? If possible run with extra verbosity (-vvvv) and diff (--diff) -->
<!-- Please also include check mode (--check --diff) output if the API returns an error -->
<!-- Be sure to mask any sensitive information -->
Connector failed to start with the following exception logged in catalina.out:

```
SEVERE [main] org.apache.catalina.util.LifecycleBase.handleSubClassException Failed to initialize component [Connector[HTTP/1.1-8567]]
        org.apache.catalina.LifecycleException: Protocol handler initialization failed
                at org.apache.catalina.connector.Connector.initInternal(Connector.java:1049)
                at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:136)
                at org.apache.catalina.core.StandardService.initInternal(StandardService.java:561)
                at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:136)
                at org.apache.catalina.core.StandardServer.initInternal(StandardServer.java:1049)
                at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:136)
                at org.apache.catalina.startup.Catalina.load(Catalina.java:724)
                at org.apache.catalina.startup.Catalina.load(Catalina.java:746)
                at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                at java.lang.reflect.Method.invoke(Method.java:498)
                at org.apache.catalina.startup.Bootstrap.load(Bootstrap.java:305)
                at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:475)
        Caused by: java.net.SocketException: Permission denied
                at sun.nio.ch.Net.bind0(Native Method)
                at sun.nio.ch.Net.bind(Net.java:461)
                at sun.nio.ch.Net.bind(Net.java:453)
                at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:222)
                at org.apache.tomcat.util.net.NioEndpoint.initServerSocket(NioEndpoint.java:272)
                at org.apache.tomcat.util.net.NioEndpoint.bind(NioEndpoint.java:227)
                at org.apache.tomcat.util.net.AbstractEndpoint.bindWithCleanup(AbstractEndpoint.java:1208)
                at org.apache.tomcat.util.net.AbstractEndpoint.init(AbstractEndpoint.java:1221)
                at org.apache.coyote.AbstractProtocol.init(AbstractProtocol.java:603)
                at org.apache.coyote.http11.AbstractHttp11Protocol.init(AbstractHttp11Protocol.java:80)
                at org.apache.catalina.connector.Connector.initInternal(Connector.java:1046)
                ... 13 more
```

--- a/roles/jws/tasks/systemd/selinux.yml
+++ b/roles/jws/tasks/systemd/selinux.yml
@@ -26,3 +26,14 @@
         - "Selinux policy created"
         - "Apply selinux policy"
   when: archive_path_selinux.stat.exists
+
+- name: Allow tomcat to listen to port
+  community.general.seport:
+    ports:
+      - "{{ tomcat_listen_http_port }}"
+      - "{{ tomcat_listen_https_port }}"
+      - "{{ tomcat_listen_ajp_port }}"
+      - "{{ tomcat_shutdown_port }}"
+    proto: tcp
+    setype: http_port_t
+    state: present

